modules/setting/setting.go

```diff
@@ -732,11 +732,16 @@ func NewContext() {
 	SSH.AuthorizedKeysBackup = sec.Key("SSH_AUTHORIZED_KEYS_BACKUP").MustBool(true)
 	SSH.ExposeAnonymous = sec.Key("SSH_EXPOSE_ANONYMOUS").MustBool(false)
 
-	if err = Cfg.Section("server").MapTo(&LFS); err != nil {
+	sec = Cfg.Section("server")
+	if err = sec.MapTo(&LFS); err != nil {
 		log.Fatal(4, "Failed to map LFS settings: %v", err)
 	}
+	LFS.ContentPath = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(AppDataPath, "lfs"))
```

Comment:
either move this line after `if LFS.StartServer {` or move `if ...IsAbs(...` after this line so that both these lines are together

Reply:
👍

Reply:
Set it like this instead: https://github.com/cez81/gitea/blob/d766d0c4e064bf7f66098123f39d15c2dc67e415/modules/setting/setting.go#L116
```go
}{
  ContentPath: "data/lfs"
}
```
As seen for example here: https://github.com/cez81/gitea/blob/d766d0c4e064bf7f66098123f39d15c2dc67e415/modules/setting/setting.go#L214

Reply:
we really need to decide how to specify default value in code. We have at least two different ways to do that that is not very good :)

Reply:
I think setting it like this is better. Don't want to hard code the data folder, right?